pubsub/v1samples/subscriptions/create_message_transforms.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package subscriptions
+
+// [START pubsub_old_version_create_subscription_with_smt]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func createSubscriptionWithSMT(w io.Writer, projectID, subID string, topic *pubsub.Topic) error {
```

Comment:
nit: I've started asking contributors to provide a short comment describing what the sample function does and what each of the arguments is.

Reply:
Yeah that makes sense. I added a comment to both new sample functions.